File: PHP/0001.php
<?php
// Function to calculate the sum of multiples of 3 or 5 below a given limit
function sum_of_multiples($limit) {
    $sum = 0;

    for ($i = 1; $i < $limit; $i++) {
        if ($i % 3 == 0 || $i % 5 == 0) {
            $sum += $i;
        }
    }

    return $sum;
}

// Define the limit
$limit = 1000;

// Calculate the result
$result = sum_of_multiples($limit);

// Print the result
echo "The sum of all multiples of 3 or 5 below $limit is: $result\n";
?>

File: Lua/0001.lua
function sumOfMultiples(limit)
    local sum = 0
    for i = 1, limit - 1 do
        if i % 3 == 0 or i % 5 == 0 then
            sum = sum + i
        end
    end
    return sum
end

local limit = 1000

local result = sumOfMultiples(limit)
print("The sum of all multiples of 3 or 5 below " .. limit .. " is: " .. result)